topo/node/keysight/keysight.go

```diff
@@ -322,6 +322,12 @@ func (n *Node) FixInterfaces() {
 	}
 }
 
+func (n *Node) HardwareLoopbackSupported() bool {
```

Comment:
After discussion with Alex we feel this is might not be the best name for this feature.  I am trying to think of better terminology but the only thing I have come up with us "BackToBack" vs "HardwareLoopback", but I am not at all set on that name.  Any other suggested names?  We can then look at the suggested names and select one.  LoopbackCable?  LoopbackLink?

Reply:
- DirectLoopback
- SelfLoop
- SelfLoopback

I don't know what this situation is called on real hardware routers so I'll defer to you

Reply:
settled on BackToBackLoop offline